packages/edge-gateway/src/gateway.js

```diff
@@ -33,7 +47,7 @@ export async function gatewayGet(request, env) {
  *
  * @param {Response} response
  */
-function getTransformedResponseWithCustomHeaders(response) {
+function getTransformedResponseWithCspHeaders(response) {
   const clonedResponse = new Response(response.body, response)
 
   clonedResponse.headers.set(
```

Comment:
maybe this is the plan for a future PR, but I feel like the list of allowed `connect-src` and maybe the CSP more generally should be driven by the tags in the goodbits list. Like "CID `x` needs to connect to url `y` etc.

Reply:
We did not talk about this yet. I have that as a question mark on my head as well. We will figure out later as use cases appear :)